Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -102,6 +102,7 @@ class ActivityStreamPanel: UICollectionViewController, HomePanel {
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
         coordinator.animate(alongsideTransition: {context in
+            self.presentedViewController?.dismiss(animated: true, completion: nil)
```

Comment:
This should fix the rotation issues. Dismissing the menu when a rotation happens.